payments_service/auth/tests/test_views.py

```diff
@@ -30,6 +31,12 @@ def set_solitude_buyer(self):
         self.solitude.generic.buyer.get_object.return_value = buyer
         return buyer
 
+    def set_solitude_buyer_patcher(self):
+        # Set up support for solitude.generic.buyer(user_id).patch(...)
+        buyer = mock.Mock()
+        self.solitude.generic.buyer.return_value = buyer
+        return buyer.patch
```

Comment:
I feel mocks are going to be the death of me, but I know that isn't helping so just tell me to shut the heck up.